addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/api/HueState.java

```diff
@@ -80,4 +84,13 @@ public String toString() {
         return "[on: " + on + " bri: " + bri + " hue: " + hue + " sat: " + sat + " xy: " + xyString + " ct: " + ct
                 + " alert: " + alert + " effect: " + effect + " colormode: " + colormode + " reachable: " + reachable;
     }
+
+    public boolean isOn() {
+      // thelight is on if bolean is passed or we have a brightness
```

Comment:
typo 'thelight', 'bolean'

consider also fixing the grammar:
```java
// the light is on if a boolean is passed or if we have a brightness
```

Please note that code checks the boolean `this.on` so to me it not directly clear what you mean with `thelight is on if bolean is passed`, that context is not present in this method.

Reply:
Can not think of a better explanation.
On is the Boolean received. So the Light is in State on if this on variable holds true.
I wouldn't write "The light is on if on is true" but you are right, reading it again it would be more like
"The Light is on if Boolean variable received is true or......"